src/Sequence.tsx

```diff
@@ -0,0 +1,25 @@
+import { fr } from "@codegouvfr/react-dsfr";
+import { useColors } from "@codegouvfr/react-dsfr/useColors";
+import { Declarations } from "@inseefr/lunatic";
+import { DeclarationType } from "./utils/type/type-source";
+type SequenceType = {
+    label: string;
+    declarations: Array<DeclarationType>;
+};
+
+const afterPostion = "AFTER_QUESTION_TEXT";
+
+export function Sequence({ label, declarations }: SequenceType) {
+    const theme = useColors();
+    return (
+        <div
+            className={fr.cx("fr-grid-row", "fr-p-4w")}
```

Comment:
Maybe we can add class `"sequence-lunatic-dsfr"` so that we can easily identify the component in the inspector, and so that it's consistent with Lunatic (see [here](https://github.com/InseeFr/Lunatic/blob/f87bd39ae679e80541703db2f9331e493fd7a077/src/components/sequence/sequence.jsx#L14))